constants.py

```diff
@@ -24,6 +24,7 @@
 # Where XX is the sensor_id
 FM_GET_DATA_URL = FM_BASE_URL + "dev/sensor/"
 FM_GET_DATA_SLUG = "/chart_data/"
+FM_CHARGING_COST_SENSOR_ID = 76
```

Comment:
Will change this to a user setting. It does make setup a tiny bit more difficult.. Sort of reminder that it would be good to get this from FM via API...